packages/react-instantsearch/src/core/createInstantSearchManager.js

```diff
@@ -41,6 +41,11 @@ export default function createInstantSearchManager({
     searching: false,
   });
 
+  function updateClient(client) {
+    helper.setClient(client);
+    search(); // Will it work?
```

Comment:
Just wanted to be sure because we've discussions about related topics. Fine with you @mthuret ?